### ISSUE TITLE
Solaris build is broken 💥

### ISSUE THREAD

**Issue by hairyhenderson** [hairyhenderson/gomplate#255]
🤦‍♂️

This was introduced somewhere between v2.2.0 and now: https://github.com/hairyhenderson/gomplate/compare/v2.2.0...master

```console
$ GOOS=solaris GOARCH=amd64 CGO_ENABLED=0 go build -ldflags "-w -s"                                          
# github.com/hairyhenderson/gomplate/vendor/golang.org/x/sys/unix
vendor/golang.org/x/sys/unix/zsyscall_solaris_amd64.go:13:3: //go:cgo_import_dynamic libc_pipe pipe "libc.so" only allowed in cgo-generated code
vendor/golang.org/x/sys/unix/zsyscall_solaris_amd64.go:14:3: //go:cgo_import_dynamic libc_getsockname getsockname "libsocket.so" only allowed in cgo-generated code
vendor/golang.org/x/sys/unix/zsyscall_solaris_amd64.go:15:3: //go:cgo_import_dynamic libc_getcwd getcwd "libc.so" only allowed in cgo-generated code
vendor/golang.org/x/sys/unix/zsyscall_solaris_amd64.go:16:3: //go:cgo_import_dynamic libc_getgroups getgroups "libc.so" only allowed in cgo-generated code
vendor/golang.org/x/sys/unix/zsyscall_solaris_amd64.go:17:3: //go:cgo_import_dynamic libc_setgroups setgroups "libc.so" only allowed in cgo-generated code
vendor/golang.org/x/sys/unix/zsyscall_solaris_amd64.go:18:3: //go:cgo_import_dynamic libc_wait4 wait4 "libc.so" only allowed in cgo-generated code
vendor/golang.org/x/sys/unix/zsyscall_solaris_amd64.go:19:3: //go:cgo_import_dynamic libc_gethostname gethostname "libc.so" only allowed in cgo-generated code
vendor/golang.org/x/sys/unix/zsyscall_solaris_amd64.go:20:3: //go:cgo_import_dynamic libc_utimes utimes "libc.so" only allowed in cgo-generated code
vendor/golang.org/x/sys/unix/zsyscall_solaris_amd64.go:21:3: //go:cgo_import_dynamic libc_utimensat utimensat "libc.so" only allowed in cgo-generated code
vendor/golang.org/x/sys/unix/zsyscall_solaris_amd64.go:22:3: //go:cgo_import_dynamic libc_fcntl fcntl "libc.so" only allowed in cgo-generated code
vendor/golang.org/x/sys/unix/zsyscall_solaris_amd64.go:22:3: too many errors
```

**Comment by hairyhenderson** [comment]
Looks like this has nothing to do with gomplate code, but with Go 1.9.4 - specifically the fix for https://github.com/golang/go/issues/23672... I'll build a Solaris binary with Go 1.9.3 for gomplate v2.3.0 _for now_, but I'd rather a more permanent fix... 😉

**Comment by hairyhenderson** [comment]
This was fixed in Go 1.10, released yesterday! 🎉 The next release will be built with 1.10, so no worries there.

The fix was https://go-review.googlesource.com/c/go/+/94018 - as mentioned in https://github.com/golang/go/issues/23839
